contracts/contracts/sbtc-registry.clar

```diff
@@ -319,8 +328,36 @@
   )
 )
 
-;; Private functions
+;; Update protocol contract
+;; This function can only be called by the active bootstrap-signers contract
+(define-public (update-protocol-contract
```

Comment:
Using a map for protocol contracts (instead of the single var) would also really simplify this function and `validate-protocol-caller` 😄